src/frontend/komponenter/Fagsak/Søknad/RegistrerSøknad.tsx

```diff
@@ -96,33 +98,48 @@ const RegistrerSøknad: React.FunctionComponent = () => {
             {feilmelding && <Feilmelding children={feilmelding} />}
             <div style={{ display: 'flex' }}>
                 <div style={{ flex: 1 }} />
-                <Knapp
-                    onClick={() => {
-                        if (fagsak.status === RessursStatus.SUKSESS && erSøknadGyldig()) {
-                            const aktivBehandling = hentAktivBehandlingPåFagsak(fagsak.data);
-                            settSenderInn(true);
-
-                            axiosRequest<IFagsak, ISøknadDTO>({
-                                method: 'POST',
-                                data: søknad,
-                                url: `/familie-ba-sak/api/behandlinger/${aktivBehandling?.behandlingId}/registrere-søknad-og-hent-persongrunnlag`,
-                            }).then((response: Ressurs<IFagsak>) => {
-                                settSenderInn(false);
-                                if (response.status === RessursStatus.SUKSESS) {
-                                    settFagsak(response);
-
-                                    history.push(`/fagsak/${response.data.id}/vilkaarsvurdering`);
-                                } else if (response.status === RessursStatus.FEILET) {
-                                    settFeilmelding(response.melding);
-                                } else {
-                                    settFeilmelding('Registrering av søknaden feilet');
-                                }
-                            });
-                        }
-                    }}
-                    children={'Bekreft og fortsett'}
-                    spinner={senderInn}
-                />
+                {erLesevisning() ? (
+                    <Hovedknapp
```

Comment:
Litt spesiell håndtering av navigasjonsknapper her. Camilla og Gunn skulle ta en gjennongang av hele navigasjonen, så det ble konsistent mtp type knapper og tekst. 

Kanskje man også her bør bruke generelle Skjemasteg, som i Vilkårsvurdering.tsx.

Reply:
Jeg stemmer for å skrive om til å bruke Skjemasteg 🙌